packages/@okta/vuepress-site/docs/guides/sign-users-out/before-you-begin/index.md

```diff
@@ -1,9 +1,16 @@
 ---
 title: Before you begin
 ---
+
+Signing users out is an important part of security as it helps prevent unauthorized actions on a current session. You can sign a user out of both the session inside your application and the Okta session.
```

Comment:
I would change "You can sign a user out of both the session inside your application and the Okta session." to something like:

> To sign a user out of your application, you will usually need to close both their session in your application as well as their Okta session.

Reply:
This is kind of a tricky statement as it mostly applies to server-side and mobile applications, but only _sort of_ applies to things like Single Page Apps. The main point I'd want to get across is that you _always_ need to end the Okta session, and _usually_ need to end the application session.

There's also the issue of federated applications. Closing an Okta session will *not* end other application sessions. Perhaps the best way to frame this would be to say something on the order of:

> An important part of security is to minimize the chances that a malicious actor uses an existing session to perform unauthorized actions. The most common strategies to prevent unauthorized use of a session include setting short token lifetimes and giving users the ability to sign out after they are done. This guide explains how to sign users out of an application.
>
> Signing users out of an application secured using Okta requires that you close users session in Okta. And in the cases where your application also has session you will also need to close the application session as well.

If you end up using text like the above, you'd want to link to the token lifetimes section above and also change the order of the bullet points below.